src/index.tsx

```diff
@@ -20,22 +18,22 @@ export const load = (environment: 'prod' | 'sandbox'): Promise<void> =>
           resolve()
         }
       )
-      // $FlowFixMe
+
       document.body.appendChild(s)
     }
   )
 
 type DwollaIAVError = {
-  code: string,
-  message: string,
+  code: string
+  message: string
```

Comment:
no commas here in TS?

Reply:
I guess not, Prettier did this.

Reply:
no. the options are semi colons or nothing. it's weird.